test/scripts/download-latest-rt-binaries.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+
+# Exit on any error
+set -e
+
+# Always run the commands from the "test" dir
+cd $(dirname $0)/..
+
+LATEST_RUNTIME_RELEASE=$(curl -s https://api.github.com/repos/moondance-labs/tanssi/releases | jq -r '.[] | select(.name | test("runtime";"i")) | .tag_name' | head -n 1 | tr -d '[:blank:]') && [[ ! -z "${LATEST_RUNTIME_RELEASE}" ]]
```

Comment:
I bet this won't work because jq is not installed in the runner (but maybe I'm wrong)

Reply:
it should, it's the same we are doing in the CI through a script

Reply:
https://github.com/moondance-labs/tanssi/blob/a8336a40e4fe65e8d6f907002f7ef69e1885e9d4/.github/workflows/release.yml#L55